### PR TITLE
Added 'M' to the number formatter

### PR DESCRIPTION
**Pull request by RaceProUK**

### CODE REVIEW

**Comment by RaceProUK** [comment]
I would sign the CLA, but the URL just loads a blank white screen

**Comment by coding-horror** [comment]
Loads ok in Mobile Safari for me..

**Comment by RaceProUK** [comment]
Works in IE too; must be a Firefox thing.

Anyway, CLA signed :)

**Comment by SamSaffron** [comment]
seems fine, thanks!

**Comment by coding-horror** [comment]
There is a little weird white-screen pause in Firefox while it does.. god knows what.. but it does load.

image

File: app/assets/javascripts/discourse/lib/formatter.js [hunk 1]
```diff
@@ -264,6 +264,9 @@ var number = function(val) {
   if (val > 999) {
     return (val / 1000).toFixed(1) + "K";
   }
+  if (val > 999999) {
```
Comment (riking) [inline app/assets/javascripts/discourse/lib/formatter.js]: Order of operations, @RaceProUK ;)

**Pull request merged**